React/Recipes/Characters encoding issue in static pages in NextJS.md

```diff
@@ -0,0 +1,57 @@
+## Motivation
+
+Although this is not an issue with NextJS but React (check more details [here](https://github.com/facebook/react/issues/13838)),
```

Comment:
I would replace "this" with the actual issue, since it's not mentioned in the paragraph at all. 

Also, as per my previous comment, do not split long lines yourself. The same goes for the rest of the file.